Compilation error in vmm/vmm_stat.c - implicit conversion changes signedness

make
cc src/vmm/x86.c
cc src/vmm/vmm.c
cc src/vmm/vmm_host.c
cc src/vmm/vmm_mem.c
cc src/vmm/vmm_lapic.c
cc src/vmm/vmm_instruction_emul.c
cc src/vmm/vmm_ioport.c
cc src/vmm/vmm_callout.c
cc src/vmm/vmm_stat.c
src/vmm/vmm_stat.c:127:29: error: implicit conversion changes signedness: 'int' to 'size_t' (aka 'unsigned long') [-Werror,-Wsign-conversion]
                                strlcpy(buf, vst->desc, bufsize);
                                ~~~~~~~                 ^~~~~~~
1 error generated.
Makefile:88: recipe for target 'build/vmm/vmm_stat.o' failed
make: *** [build/vmm/vmm_stat.o] Error 1

cc -v
Apple LLVM version 6.1.0 (clang-602.0.53) (based on LLVM 3.6.0svn)
Target: x86_64-apple-darwin14.3.0
Thread model: posix

--- a/include/xhyve/vmm/vmm_stat.h
+++ b/include/xhyve/vmm/vmm_stat.h
@@ -93,7 +93,7 @@ void 	vmm_stat_free(void *vp);
  * 'buf' should be at least fit 'MAX_VMM_STAT_TYPES' entries
  */
 int	vmm_stat_copy(struct vm *vm, int vcpu, int *num_stats, uint64_t *buf);
-int	vmm_stat_desc_copy(int index, char *buf, int buflen);
+int	vmm_stat_desc_copy(int index, char *buf, size_t buflen);
 
 static void __inline
 vmm_stat_array_incr(struct vm *vm, int vcpu, struct vmm_stat_type *vst,

--- a/src/vmm/vmm_stat.c
+++ b/src/vmm/vmm_stat.c
@@ -112,7 +112,7 @@ vmm_stat_free(void *vp)
 }
 
 int
-vmm_stat_desc_copy(int index, char *buf, int bufsize)
+vmm_stat_desc_copy(int index, char *buf, size_t bufsize)
 {
 	int i;
 	struct vmm_stat_type *vst;

